src/components/Comment.svelte

```diff
@@ -99,17 +124,39 @@
     {:else}
       <Markdown {rawPath} content={body} />
     {/if}
-    {#if groupedReactions.size > 0}
-      <div class="reactions">
-        {#each groupedReactions as [reaction, nids], key}
-          <Chip {key}>
-            <div class="reaction">
+    <div class="reactions">
+      {#if $httpdStore.state === "authenticated"}
```

Comment:
Can we move this check one element above so it wraps the reaction container?

Reply:
I think we should keep it like this, so we can show the reactions to a comment, even if a user isn't authenticated.